### ISSUE TITLE
What is the model saved by the trainer by default?

### ISSUE THREAD

**Issue by Hans0124SG** [PyTorchLightning/pytorch-lightning#2215]
## ❓ Questions and Help

Thanks for the great library!
May I know that which model will the trainer use when we run `trainer.test()`  by default?
Is it the best model during training that has the lowest validation loss?

**Comment by rohitgr7** [comment]
Yeah the best one
https://github.com/PyTorchLightning/pytorch-lightning/blob/1635ba1bb38e69d985b9a96e1e8f6b6dbdeff268/pytorch_lightning/trainer/trainer.py#L1076-L1081

**Comment by Hans0124SG** [comment]
Thank you very much!
